packages/lodestar/package.json

```diff
@@ -0,0 +1,120 @@
+{
+  "name": "@chainsafe/lodestar",
+  "version": "0.1.0",
+  "description": "A Typescript implementation of the beacon chain and validator client",
+  "author": "ChainSafe Systems",
+  "license": "GPL-3.0",
+  "bugs": {
+    "url": "https://github.com/ChainSafe/lodestar/issues"
+  },
+  "homepage": "https://github.com/ChainSafe/lodestar#readme",
+  "main": "lib/index.js",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "build": "babel packages/beaconChain -x .ts -d lib --source-maps",
+    "build:docs": "typedoc --exclude packages/beaconChain/index.ts --out docs src",
+    "check-types": "tsc --noEmit --incremental",
+    "lint": "eslint --ext .ts packages/beaconChain/",
+    "lint-fix": "eslint --ext .ts packages/beaconChain/ --fix",
+    "pretest": "yarn run check-types",
```

Comment:
Beacuse we will use symlinks for monorepo package dependencies, everything needs to be builded before executing tests

Reply:
oh interesting

Reply:
I don't remmber 100% but I think if you add build task to every package lerna can build them based on their dependency on each other